lib/tasks/storage.rake

```diff
@@ -0,0 +1,17 @@
+require 'active_storage/migration'
```

Comment:
I know this isn't necessary for this task, but it seemed list a waste to throw away. Perhaps it'll be useful in the future or for another project. It's a simple rake task that will allow migrating Active Storage data from local to Amazon, or vice versa:

```
bundle exec rake active_storage:aws_to_local
bundle exec rake active_storage:local_to_aws
```